dominion-solutions/n8n
# Ethan sowell/cu c0ry5u/build a clockify writer for time entries

# Details
Creates time entries using the ClockifyWriter

# Steps to test changes

1. Use npm run dev to start the localhost.
1. Add a Clockify regular node.
1. Select "Create" and "Time Entry" and put in all the information required, then click execute.
1. Make sure information returned is right and that a Entry is made.
1. Now, make  a Clockify Trigger node and enter API key and info for a workspace.
1. Execute it to pull information from the other workspace
1. Drag the arrow to add another Clockify regular node to it.
1. Select "Create and "Time Entry" and put in all the information required, including expressions to the pulled info from the trigger.
1. Execute it and make sure all the Time Entries are returned and created properly.

# Issues Addressed

[CU-c0ry5u](https://app.clickup.com/t/c0ry5u) - Build a Clockify Writer for Time Entries
